src/Microsoft.Data.SqlClient/tests/tools/TDS/TDS.Servers/GenericTDSServer.cs

```diff
@@ -113,6 +113,12 @@ public virtual TDSMessageCollection OnPreLoginRequest(ITDSServerSession session,
             // Generate server response for encryption
             TDSPreLoginTokenEncryptionType serverResponse = TDSUtilities.GetEncryptionResponse(preLoginRequest.Encryption, Arguments.Encryption);
 
+            //TODO: Add flag for client certificate authentification if its requested
+            //if(ClientCertificate)
+            //{
+            //serverResponse |= TDSPreLoginTokenEncryptionType.ClientCertificateOn;
```

Comment:
Why is this commented?

Reply:
Because i was not sure if i need to force CertificateOn or CertificateOff
where difference is CertificateOff is without encryption